extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml

```diff
@@ -724,7 +724,7 @@
             <name>ActivityTagsFeatureProperties</name>
             <description>New Activity Stream Feature enablement flag</description>
             <property name="exo.feature.ActivityTags.enabled" value="${exo.feature.ActivityTags.enabled:false}"/>
```

Comment:
IMO, you just need to remove the permission and set the default value to true

Reply:
I agree . Done!